src/battle_main.c

```diff
@@ -4688,10 +4689,41 @@ static void CheckMegaEvolutionBeforeTurn(void)
         }
     }
 
+    #if B_MEGA_EVO_ALTER_TURN_ORDER <= GEN_6
```

Comment:
```suggestion
    #if B_MEGA_EVO_TURN_ORDER <= GEN_6
```

Reply:
Since it doesn't determine in which order the mega evolution will happen, but rather that, when happening, they may alter the turn order, I believe the first name is better

Reply:
Both labels are the same though? It's literally just one less word that isn't needed, as the effect of the config is more properly explained in the comment.